src/pages/ProvideLiquidity/LiquidityCard.tsx

```diff
@@ -0,0 +1,91 @@
+import { Token } from '@ubeswap/sdk'
+import { PoolCard } from 'components/earn/PoolCard'
+import QuestionHelper from 'components/QuestionHelper'
+import { RowBetween, RowFixed } from 'components/Row'
+import { CompoundBotSummaryBase } from 'pages/Compound/useCompoundRegistry'
+import { useLPValue } from 'pages/Earn/useLPValue'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+import { TYPE } from 'theme'
+
+import AddLiquidityConfirm from './AddLiquidityConfirm'
+import AddLiquidityForm from './AddLiquidityForm'
+
+const Container = styled.div<{ $expanded: boolean }>`
+  margin-top: ${({ $expanded }) => ($expanded ? '12px' : '0')};
+  max-height: ${({ $expanded }) => ($expanded ? '610px' : '0')};
+  transition: all 0.2s ${({ $expanded }) => ($expanded ? 'ease-in' : 'ease-out')};
+`
+
+interface Props {
+  token0: Token
+  token1: Token
+  compoundBotSummary: CompoundBotSummaryBase
```

Comment:
here's a place where "compoundBot" naming is used

Reply:
i've gone through and renamed them just now